app/models/agents/liquid_output_agent.rb

```diff
@@ -0,0 +1,214 @@
+module Agents
+  class LiquidOutputAgent < Agent
+    include WebRequestConcern
```

Comment:
That seems true. But did I put it in there to get it to be accessible to the front end? I can't remember. 

I do remember that I started this agent from a copy of another agent, so this might be a remenant of that.